changelogs/fragments/8199-added-usb-support-to-proxmox-module.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - "proxmox_kvm - adds``usb`` parameter for setting usb devices on proxmox KVM VMs (https://github.com/ansible-collections/community.general/pull/8199)."
```

Sorry, I missed that yesterday. I'll just push it and then merge once CI passed.
```suggestion
  - "proxmox_kvm - adds``usb`` parameter for setting USB devices on proxmox KVM VMs (https://github.com/ansible-collections/community.general/pull/8199)."
```